ios/Exponent/ExpoKit/ExpoKit.m

```diff
@@ -112,9 +112,10 @@ - (void)application:(UIApplication *)application didFinishLaunchingWithOptions:(
       }
     }
   }
-
+  
   // This is safe to call; if the app doesn't have permission to display user-facing notifications
   // then registering for a push token is a no-op
```

Comment:
This comment applies to the remote notification manager code, keep the lines together